metrics/grafana/common.py

```diff
@@ -592,14 +627,32 @@ def expr_histogram_avg(
 
 
 def target(
-    expr: Union[Expr, str],
+    expr: Union[Expr, OpExpr, str],
     legend_format: Optional[str] = None,
     hide=False,
     data_source=DATASOURCE,
     interval_factor=1,  # Prefer "high" resolution
+    # It indicates whether to add additional groupby label to the groupby and legend.
+    # If the expr is already groupby by instance, no need to set it to True.
+    # Additional groupby is used to support optional by instance. By default, it doesn't
+    # take effect. When the variable $additional_groupby of dashboard is set to `instance`.
+    # The expr would be groupby by `instance`.
+    additional_groupby=False,
```

Comment:
How about enable it by default?

Reply:
That would make the query too heavy. For the queries already being by instance, `additional_groupby=false` doesn't affect them.